pkg/querier/queryrange/queryrange.proto

```diff
@@ -46,7 +47,8 @@ message CachedResponse  {
 
 message Extent  {
 	int64 start = 1 [(gogoproto.jsontag) = "start"];
-	int64 end = 2 [(gogoproto.jsontag) = "end"];
-  APIResponse response = 3 [(gogoproto.jsontag) = "response"];
+  int64 end = 2 [(gogoproto.jsontag) = "end"];
+  reserved 3;
```

Comment:
Why is this reserved?

Reply:
This is to ease the cache transition. We ignore the old response.

Reply:
Hrm, curious how it's done. Could you also add a comment to make it clear to future readers?

Reply:
well it simple, since it's reserved and the new property uses another key index, the new code won't catch old entry values and so update the cache accordingly.

I will add a comment.